test/custom_rules/__init__.py

```diff
@@ -0,0 +1 @@
+"""Dummy test module."""
```

Comment:
Instead of creating the fixture folders in-tree, it's better to use a `tmp_path` fixture that comes built-in with pytest: https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture.
It'll help you have a temporary directory where you'd write anything needed for the test.

You'd probably want to make your own fixture-function to manage this. e.g. https://github.com/ansible/ansible-lint/blob/66870af7758a76b3dc87464306bb5410ab0bd257/test/TestSkipImportPlaybook.py#L24-L32

Reply:
Thanks for your advice, but, IMHO, it needs @pytest.fixture + @pytest.mark.parametrize combination (e.g. indirect parameterization) and that causes too complex code. So, I want to keep as it is now.